Resources/Public/Javascript/PageView/FulltextControl.js

```diff
@@ -433,6 +433,10 @@ dlfViewerFullTextControl.prototype.enableFulltextSelect = function() {
         .attr('title', this.dic['fulltext-off']);
     }
 
+    if(this.dic['launch-state'] === 'off') {
+        $("#tx-dlf-tools-fulltext").css('visibility', 'hidden');
+    }
+
```

Comment:
This should be removed. We want to disable the link only in the Zeitungsportal, so this should be added to some custom CSS.

Reply:
I have understood that `off` is a custom thing for Zeitungsportal, so if the full text is always visible (off) the link should be invisble. If setting is set up to `on`, it should be all like before.

Reply:
There are actually three different cases:

* If someone wants to have fulltexts initially disabled when loading a page, but available to be turned on by the user: `launchState = off`.
* If someone wants to have fulltexts initially shown when loading a page, but possible for the user to turn it off: `launchState = on`.

Those two general cases essentially work the same: Fulltext can be switched on and off by a user. The only difference is, if the fulltext is initially turned on or off when loading the page and without any user interaction.

Then there is a third case, which we only need for the Zeitungsportal:

* We want the fulltext to be turned on by default (`launchState = on`), but the user shouldn't be able to turn it off. We don't need any code in Kitodo.Presentation to achieve this, as we can simply use the second option from above and hide the link. The latter is easy, since we can add some custom CSS to the page or just omit the placeholder from the tool's template.

Reply:
Oh, I have understood it a bit differently. So actually it needs additional checkbox which is visible in the frontend and Zeitungsportal needs on their side hide it by css. Also logic of the switches needs to be reveresed.

Reply:
The checkbox isn't needed since the `launchState` in Typoscript is sufficient, but otherwise you're right.

Reply:
Checkbox removed, logic reveresd, clicking on 'Volltext' makes nothing as all click actions are removed from this text.